src/libtorch.cc

```diff
@@ -169,7 +173,16 @@ ModelState::LoadModel(
 
   // InferenceMode should be used to guard all tensors operations including
   // model loading: https://pytorch.org/cppdocs/notes/inference_mode.html
-  torch::InferenceMode infer_guard(InferenceMode());
+  torch::InferenceMode infer_guard(EnabledInferenceMode());
+
+  // NvFuser is only supported for GPU instances
```

Comment:
When nvfuser becomes the default in pytorch, setting ENABLE_NVFUSER to false will not disable it. I think we need to have a 3-way setting for it (could use std::optional) so that we can know if it was explicitly set to true or false or not set at all. Then we can explicitly enable or disable or do nothing. PyTorch team mentioned that they are working with upstream to make it the default and it seems like being able to explicitly have it on or off may be important.